### PR TITLE
Updating build information

### PR DESCRIPTION
**Pull request by DylanHBL**
Updated documentation following an internal review.

### CODE REVIEW

File: source/build/citrix.md [hunk 1]
```diff
@@ -12,15 +12,15 @@ Below outlines how the proposed configuration will be configured for the Citrix
 | CTXADC01 / CTXNS | Citrix Netscaler ADC       | Citrix ADC Appliance                   |
 +------------------+----------------------------+----------------------------------------+
 | CTXSF01          | Citrix StoreFront          | Citrix Storefront                      |
-|                  |                            | Citrix Receiver                        |
-|                  |                            | IIS                                    |
+|                  |                            | | Citrix Receiver                      |
```
Comment (terencesimms) [inline source/build/citrix.md:15]: image
 Is this how you want it to look? Are they sub-roles?

File: source/build/enterprise_updates.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Updates
 ```eval_rst
 .. seealso::
-   Please note that updates can be customised using your Enterprise Launch Questionnaire.
+   If you require a custom update configuration, please raise with the Solutions Architect who's designing your solution.
```
Comment (terencesimms) [inline source/build/enterprise_updates.md]: can remove 'who's' as not required.

File: source/build/ssl_certificates.md [hunk 3]
```diff
@@ -5,14 +5,18 @@
 ```
 We offer a range of certificates from Single Domain, Multiple Domain to Wildcard Certificates each ranging from Domain Control Validated to Extended Validated certificates.
 
+**Single-Domain SSLs**
```
Comment (terencesimms) [inline source/build/ssl_certificates.md:8]: Two things for this..

1. is this a duplicate of https://docs.ukfast.co.uk/domains/ssl/types.html?
2. I would remove the tables, as they don't add or clarify anything - rather they seem to duplicate the bullet points.

File: source/build/ssl_certificates.md [hunk 4]
```diff
@@ -24,7 +28,21 @@ We offer a range of certificates from Single Domain, Multiple Domain to Wildcard
 | Extended Validation Multi-Domain SSL |
 +--------------------------------------+
 ```
+- Multi-Domain SSL -  3 domains are included in the standard price, and additional domains can be added at a lower price than a single domain SSL.
+- Unified Communications SSL - Users of Microsoft Exchange 2010/2013 or Office Communications Server 2007 are able to consolidate all certificates into a single, manageable UC Certificate.
+- Extended Validation Multi-Domain SSL -  Like regular EV certificates (see below), EV Multi Domain SSLs also display the green HTTPS website address bar to visitors. 3 domains are included in the standard price and additional domains can be added at a lower price than a single domain SSL.
```
Comment (terencesimms) [inline source/build/ssl_certificates.md:16]: Not sure this is true anymore (the green bar) - certainly being moved away from

File: source/build/ukfast_backups.md [hunk 5]
```diff
@@ -26,13 +26,12 @@ In order for UKFast to simplify our backup schedule offering without losing the
...
 
-When referring to Database Backups by default we will apply the below configurations to the following Backups Agents: 
--	Microsoft Exchange 
--	Microsoft SharePoint 
+When referring to Database Backups by default we will apply the below configurations to the following Backups Agents:  
 -	Microsoft SQL Server
 -	PostgreSQL
 -	Oracle DB 
 -	MySQL Server / MariaDB
+-  MongoDB
```
Comment (terencesimms) [inline source/build/ukfast_backups.md]: If commvault, worth confirming if this is actually working, as last we checked Commvault had only set up one client as was called 'broken' or something

**Review by terencesimms — CHANGES_REQUESTED** [review]
just a few changes, loads done here!

File: source/build/citrix.md (hunk 1 above)
Comment (DylanHBL) [reply source/build/citrix.md:15]: I read that an extra line with put them under each other, as currently it just puts them all on the same line.

File: source/build/enterprise_updates.md (hunk 2 above)
Comment (DylanHBL) [reply source/build/enterprise_updates.md]: Done.

File: source/build/ssl_certificates.md (hunk 3 above)
Comment (DylanHBL) [reply source/build/ssl_certificates.md:8]: Potentially has the same content, however this is just for build docs so we need a section on SSLs without linking outside. Agreed, I've removed the tables.

File: source/build/ssl_certificates.md (hunk 4 above)
Comment (DylanHBL) [reply source/build/ssl_certificates.md:16]: I think I pulled this information from the MyUKFast area when you go to buy an SSL, so it matches what they'll see if they went there.

File: source/build/ukfast_backups.md (hunk 5 above)
Comment (DylanHBL) [reply source/build/ukfast_backups.md]: Removed until CV send me feedback.

**Review by DylanHBL — COMMENTED** [review]
Updated following feedback.

**Review by terencesimms — APPROVED** [review]

**Pull request merged**